bundles/org.openhab.binding.bmwconnecteddrive/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="bmwconnecteddrive" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>BMW ConnectedDrive</name>
+	<description>Provides access to your Vehicle Data via BMW Connected Drive Portal</description>
+	<author>Bernd Weymann</author>
```

Comment:
The author tag is deprecated and should therefore be removed. See openhab/openhab-core#1844.

Reply:
removed